test/e2e/gang_scheduling/gang_scheduling_test.go

```diff
@@ -31,12 +31,18 @@ import (
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/common"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/k8s"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/yunikorn"
+	siCommon "github.com/apache/yunikorn-scheduler-interface/lib/go/common"
 )
 
 var _ = Describe("", func() {
 	var kClient k8s.KubeCtl //nolint
 	var restClient yunikorn.RClient
 	var ns string
+	groupA := "groupa"
+	groupB := "groupb"
+	fifoQName := "fifoq"
+	defaultPartition := "default"
+	rootNamespace := "root." + ns
```

Comment:
This is the queue based on the namespace (ns), would expect it to be called something like `nsQueue` or just `queue`. `ns` gets set in BeforeEach(). Does that not mean the value is off which causes the e2e failures?